apps/server/test/lexical/server/code_intelligence/completion/env_test.exs

```diff
@@ -739,4 +864,62 @@ defmodule Lexical.Server.CodeIntelligence.Completion.EnvTest do
                sort_items(items)
     end
   end
+
+  describe "prefix_aliases/1" do
+    test "when in current module" do
+      source = ~q<
+        defmodule User do
+          defstruct [:name, :age]
+
+          def name(%__MODULE__{|}) do
+          end
+        end
+      >
+
+      assert [:__MODULE__] == source |> new_env() |> prefix_aliases()
+    end
+
+    test "when in a current module child" do
+      source = ~q<
+        defmodule User do
+          defmodule Profile do
+            defstruct [:info]
+          end
+
+          def name(%__MODULE__.Profile{|}) do
+          end
+        end
+      >
+
+      assert [:__MODULE__, :Profile] == source |> new_env() |> prefix_aliases()
+    end
+
+    test "when in a struct" do
+      source = ~q<
+        defmodule User do
+          defstruct [:name, :age]
+
+          def name(%User{|}) do
+          end
+        end
+      >
+
+      assert [:User] == source |> new_env() |> prefix_aliases()
+    end
+
+    test "when in a struct child" do
```

Comment:
the test descriptions are pretty vague here; they sound more like something you see in a "describe". 
What are you testing? Put that in the test name.

Reply:
right, I remove them and add more tests in the `remote_control` app.